src/data/faq_de.json

```diff
@@ -485,7 +485,8 @@
                         "anchor": "other_platforms",
                         "active": true,
                         "textblock": [
-                            "Es ist im Moment nicht absehbar, wann und ob wir die App in anderen App Stores außer denen von Google oder Apple zur Verfügung stellen können. Die App wird die Apple und Google Exposure Notification APIs verwenden. Die Community kann sich gerne einbringen und die App für andere APIs übernehmen."
+                            "Es ist im Moment nicht absehbar, wann und ob wir die App offiziell in anderen App Stores außer denen von Google oder Apple zur Verfügung stellen können. Die App wird die Apple und Google Exposure Notification Frameworks verwenden. Die Community kann sich gerne einbringen und die App für andere Frameworks übernehmen.",

```

Comment:
Same as in English

Reply:
Suggest changing the historical statement from:

"Es ist im Moment nicht absehbar, wann und ob wir die App offiziell in anderen App Stores außer denen von Google oder Apple zur Verfügung stellen können. Die App wird die Apple und Google Exposure Notification Frameworks verwenden."

to:

"The Corona-Warn-App verwendet die Apple and Google Exposure Notification Framework, dementsprechend hat das Robert Koch-Institut (RKI) die App in den Apple und Google App Stores veröffentlicht. Derzeit hat das RKI keine Pläne die App in anderen Stores zur Verfügung zu stellen."

Reply:
@MikeMcC399 The suggestion sounds good, but in the text here there's a mistake, it should be "Die Corona-Warn-App" and not "The Corona-Warn-App".

Reply:
see above